### PR TITLE
Add version field in volume list command

### PR DESCRIPTION
**Pull request by RajatVaryani**
Signed-off-by: Rajat Varyani <rajat.varyani@outlook.com>

**What this PR does / why we need it**:
Adds version field in volume list command. 

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #
Fixes https://github.com/openebs/openebs/issues/1989. 

**Special notes for your reviewer**:

### CODE REVIEW

File: cmd/mayactl/app/command/volumes_list.go [hunk 1]
```diff
@@ -59,19 +59,19 @@ func (c *CmdVolumeOptions) RunVolumesList(cmd *cobra.Command) error {
...
+	out[1] = "---------|----|------|----|-------"
 	for i, items := range cvols.Items {
 		if len(items.Status.Reason) == 0 {
 			items.Status.Reason = volumeStatusOK
 		}
-		out[i+2] = fmt.Sprintf("%s|%s|%s|%s", items.ObjectMeta.Namespace,
+		out[i+2] = fmt.Sprintf("%s|%s|%s|%s|%s", items.ObjectMeta.Namespace,
 			items.ObjectMeta.Name,
-			items.Status.Reason, items.Spec.CasType)
+			items.Status.Reason, items.Spec.CasType, items.ObjectMeta.ResourceVersion)
```
Comment (sonasingh46) [inline cmd/mayactl/app/command/volumes_list.go]: @RajatVaryani 
We need not get the resource version of the object.
We need to get the openebs version for the volume.
cc @ashishranjan738

Comment (RajatVaryani) [reply cmd/mayactl/app/command/volumes_list.go]: @ashishranjan738 Can I get confirmation on what @sonasingh46 is telling?

**Comment by sonasingh46** [comment]
@RajatVaryani 
Thanks!
By any chance, you are working on this ?

File: cmd/mayactl/app/command/volumes_list.go (hunk 1 above)
Comment (ashishranjan738) [reply cmd/mayactl/app/command/volumes_list.go]: yes @RajatVaryani we want openebs volume version, not the ResourceVersion. I think it not available in the response from apiserver. You also need to make changes to the backend for getting this info. Thanks for working on this issue

**Comment by RajatVaryani** [comment]
I won't be able to work on it.

**Pull request closed without merging**